packages/server/src/http/internals/types.ts

```diff
@@ -1,15 +1,14 @@
+import { TRPCError } from '../../TRPCError';
+import { BaseHandlerOptions } from '../../internals/BaseHandlerOptions';
 import {
   AnyRouter,
-  Dict,
   ProcedureType,
-  ResponseMeta,
-  TRPCError,
   inferRouterContext,
   inferRouterError,
-} from '@trpc/server';
```

Comment:
Deno/URL imports/CDNs don't support this

Reply:
👌

Reply:
Is there a way of asserting this in a build step or something? Easy to make a mistake like this

Reply:
I can add an eslint rule for it ([no-restricted-imports](https://eslint.org/docs/rules/no-restricted-imports)).

Reply:
Do it!